sparse/unit_test/Test_Sparse_sptrsv.hpp

```diff
@@ -38,1111 +38,576 @@ using namespace KokkosKernels;
 using namespace KokkosKernels::Impl;
 using namespace KokkosKernels::Experimental;
 
-// #ifndef kokkos_complex_double
-// #define kokkos_complex_double Kokkos::complex<double>
-// #endif
-// #ifndef kokkos_complex_float
-// #define kokkos_complex_float Kokkos::complex<float>
-// #endif
-
-typedef Kokkos::complex<double> kokkos_complex_double;
-typedef Kokkos::complex<float> kokkos_complex_float;
+using kokkos_complex_double = Kokkos::complex<double>;
+using kokkos_complex_float  = Kokkos::complex<float>;
 
 namespace Test {
 
-#if 0
```

Comment:
Due to indentation change, this file is best viewed with whitespace changes hidden.